docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md

```diff
@@ -15,53 +15,63 @@ use the following commands to launch the placement translator; it
 requires the ESPW to be current at start time.
 
 ```shell
-# TODO: placement-translator needs access to multiple configs. Will remove when controllers support spaces.
-kubectl ws root:espw
-placement-translator &
-sleep 10
+(
+  export KUBECONFIG=$SM_CONFIG
+  placement-translator &
+  sleep 10
+)
 ```
 
 The following commands wait for the placement translator to get its
 job done for this example.
 
 ```shell
-# TODO: unfortunately, the $FLORIN_WS and $GUILDER_WS are mailbox ws names that we are not supporting in the space framework yet
+$FLORIN_SPACE_CONFIG="${PWD}/temp-space-config/${FLORIN_SPACE}"
+$GUILDER_SPACE_CONFIG="${PWD}/temp-space-config/${GUILDER_SPACE}"
+kubectl-kubestellar-get-config-for-space --space-name $FLOWIN_SPACE --provider-name default --sm-core-config $SM_CONFIG --output $FLORIN_SPACE_CONFIG
+kubectl-kubestellar-get-config-for-space --space-name $GUILDER_SPACE --provider-name default --sm-core-config $SM_CONFIG --output $GUILDER_SPACE_CONFIG
+
 # wait until SyncerConfig, ReplicaSets and Deployments are ready
-mbxws=($FLORIN_WS $GUILDER_WS)
+mbxws=($FLORIN_SPACE $GUILDER_SPACE)
 for ii in "${mbxws[@]}"; do
-  kubectl ws root:$ii
-  # wait for SyncerConfig resource
-  while ! kubectl get SyncerConfig the-one &> /dev/null; do
-    sleep 10
-  done
-  echo "* SyncerConfig resource exists in mailbox $ii"
-  # wait for ReplicaSet resource
-  while ! kubectl get rs &> /dev/null; do
+  (
+    export KUBECONFIG="${PWD}/temp-space-config/$ii"
+    # wait for SyncerConfig resource
+    while ! kubectl get SyncerConfig the-one &> /dev/null; do
+      sleep 10
+    done
+    echo "* SyncerConfig resource exists in mailbox $ii"
+    # wait for ReplicaSet resource
+    while ! kubectl get rs &> /dev/null; do
+      sleep 10
+    done
+    echo "* ReplicaSet resource exists in mailbox $ii"
+    # wait until ReplicaSet in mailbox
+    while ! kubectl get rs -n commonstuff commond; do
+      sleep 10
+    done
+    echo "* commonstuff ReplicaSet in mailbox $ii"
+  )
+done
+(
+  export KUBECONFIG=$GUILDER_SPACE_CONFIG
+  # check for deployment in guilder
+  while ! kubectl get deploy -A &> /dev/null; do
```

Comment:
Why the loss of specificity here? The namespace and name in the mailbox space is known statically.